build.gradle

```diff
@@ -48,6 +48,7 @@ dependencies {
     compile "org.grails:grails-plugin-services"
     compile "org.grails:grails-plugin-url-mappings"
     compile "org.grails:grails-plugin-interceptors"
+    compile "org.grails.plugins:ala-ws-security-plugin:2.0"
```

Comment:
`@RequireApiKey` and `@SkipApiKeyCheck` do not appear to be working because `shape/geojson/` and `shape/wkt/` both work without an API key.

Reply:
@adam-collins  Requests from localhost will be overpassed by ApiKeyInterceptor

![Screen Shot 2020-11-27 at 4 10 32 pm](https://user-images.githubusercontent.com/31238531/100413415-61aaa600-30cb-11eb-914d-989d5b7e1699.png)